Feature/homework 4

VtulkinVladimir opened the pull request:
Задание выполнено

VtulkinVladimir committed:
Создал приложение с коллекшн вью

реализована коллекш вью с 10 айтемами, экран с детальной информацией и дополнительный экран, открывающийся модально

VtulkinVladimir committed:
картинки для айтемов

VtulkinVladimir committed:
автофайлы

VtulkinVladimir committed:
Create contents.xcworkspacedata

VtulkinVladimir committed:
добавил проект в воркспейс

c0dedbear commented on CollectionApp/CollectionApp/Car/CarCollectionCell.swift:
```diff
@@ -0,0 +1,94 @@
+
+	
+	var car: Car? {
+		didSet {
```
Установку контента лучше вынести в отдельный метод, для didSet это слишком много кода.

c0dedbear reviewed (changes requested):
Все неплохо, но в альбомной ориентации второй экран ломается. (именно если крутить на втором экране после перехода)

Также много магических цифр в коде.

VtulkinVladimir committed:
внес изменения

Починил переворот экрана на детальном вью. теперь все вертится и меняется с переворотом
Вынес установку контента из ДидСета в отдельную функцию в CarCell
убрал немного "магических цифр"

c0dedbear reviewed (approved):
Принято
